aiida_workgraph/web/frontend/src/components/DataNodeTable.js

```diff
@@ -128,6 +183,13 @@ function DataNode() {
                 breakClassName={'pageBreak'}
             />
             <ToastContainer autoClose={3000} />
+            <WorkGraphConfirmModal
+                show={showConfirmDeleteModal}
+                setShow={setShowConfirmDeleteModal}
+                confirmAction={() => handleDeleteNode(toDeleteItem)}
+                cancelAction={() => {}}
```

Comment:
You are right that for this specific modal in this PR, the `cancelAction` is not needed but I wanted to create a more abstract modal that allows to also clean up stuff in case you allocated resources or changed some state to create the prompt. 

I don't have strong opinions about it, we can always add it back if needed, but it also does not hurt, so what do you think?

Reply:
Fine with me if you want to keep it.